### ISSUE TITLE
use line_exists - false positive with sy-tabix

### ISSUE THREAD

**Issue by FreHu** [abaplint/abaplint#1448]
These are not equivalent because line\_exists does not set sy-tabix

```abap
READ TABLE nested_tree TRANSPORTING NO FIELDS
      WITH TABLE KEY
        left = min-left.
    CHECK (  sy-subrc = 0 ).
    LOOP AT nested_tree ASSIGNING FIELD-SYMBOL(<item>) FROM sy-tabix.
      IF ( <item>-left > max-right ).
        EXIT.
      ENDIF.
      INSERT <item>-left INTO TABLE result.
    ENDLOOP.
```

```abap
CHECK line_exists( nested_tree[ left = min-left ] ).
    LOOP AT nested_tree ASSIGNING FIELD-SYMBOL(<item>) FROM sy-tabix.
      IF <item>-left > max-right.
        EXIT.
      ENDIF.
      INSERT <item>-left INTO TABLE result.
    ENDLOOP.
```

**Comment by FreHu** [comment]
Maybe as an early version don't report an issue as a precaution if sy-tabix is used within the method, or show a different message.

**Comment by larshp** [comment]
try again, 2.57.5
